catalogs/catalog-lakehouse-paimon/src/test/java/com/datastrato/gravitino/catalog/lakehouse/paimon/TestPaimonSchema.java

```diff
@@ -0,0 +1,163 @@
+    Assertions.assertEquals(ident.namespace(), schemas[0].namespace());
+
+    // clean up
+    cleanUp(paimonCatalogOperations, ident);
+  }
+
+  @Test
+  public void testAlterSchema() {
+    PaimonCatalog paimonCatalog = initPaimonCatalog("testListPaimonSchema");
+    PaimonCatalogOperations paimonCatalogOperations = (PaimonCatalogOperations) paimonCatalog.ops();
+
+    NameIdentifier ident =
+        NameIdentifier.of("metalake", paimonCatalog.name(), "test_paimon_schema");
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("key1", "val1");
+    properties.put("key2", "val2");
+
+    paimonCatalogOperations.dropSchema(ident, false);
+    PaimonSchema paimonSchema =
+        paimonCatalogOperations.createSchema(ident, COMMENT_VALUE, properties);
+    Assertions.assertTrue(paimonCatalogOperations.schemaExists(ident));
+    properties.forEach(
+        (k, v) -> Assertions.assertEquals(v, paimonSchema.toPaimonSchema().getValue().get(k)));
+
+    // schema properties of FilesystemCatalog is empty when loadDatabase.
+    Map<String, String> properties1 = paimonCatalogOperations.loadSchema(ident).properties();
+    Assertions.assertEquals(0, properties1.size());
+
+    Assertions.assertThrowsExactly(
+        UnsupportedOperationException.class,
+        () -> paimonCatalogOperations.alterSchema(ident, SchemaChange.removeProperty("key1")));
+
+    // clean up
+    cleanUp(paimonCatalogOperations, ident);
+  }
+
+  @Test
+  public void testDropSchema() {
+    PaimonCatalog paimonCatalog = initPaimonCatalog("testListPaimonSchema");
+    PaimonCatalogOperations paimonCatalogOperations = (PaimonCatalogOperations) paimonCatalog.ops();
+
+    NameIdentifier ident =
+        NameIdentifier.of("metalake", paimonCatalog.name(), "test_paimon_schema");
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("key1", "val1");
+    properties.put("key2", "val2");
+
+    paimonCatalogOperations.dropSchema(ident, false);
+    paimonCatalogOperations.createSchema(ident, COMMENT_VALUE, properties);
+    Assertions.assertTrue(paimonCatalogOperations.schemaExists(ident));
+    Assertions.assertTrue(paimonCatalogOperations.dropSchema(ident, false));
+    Assertions.assertFalse(paimonCatalogOperations.schemaExists(ident));
+
+    Assertions.assertFalse(paimonCatalogOperations.dropSchema(ident, false));
+  }
+
+  private PaimonCatalog initPaimonCatalog(String name) {
+    CatalogEntity entity =
+        CatalogEntity.builder()
+            .withId(1L)
+            .withName(name)
+            .withNamespace(Namespace.of(META_LAKE_NAME))
+            .withType(PaimonCatalog.Type.RELATIONAL)
+            .withProvider("lakehouse-paimon")
+            .withAuditInfo(AUDIT_INFO)
+            .build();
+
+    Map<String, String> conf = Maps.newHashMap();
+    conf.put(PaimonCatalogPropertiesMetadata.GRAVITINO_CATALOG_BACKEND, "filesystem");
+    conf.put(PaimonCatalogPropertiesMetadata.WAREHOUSE, "/tmp/paimon_catalog_warehouse");
```

Comment:
I'm afraid the ut may failed on windows for `/tmp/paimon_catalog_warehouse`

Reply:
ok, i will fix this.